packages/client/utils/relay/setActiveTemplate.ts

```diff
@@ -6,14 +6,14 @@ import Atmosphere from '../../Atmosphere'
 const setActiveTemplateInRelayStore = (
   store: RecordSourceProxy,
   teamId: string,
-  templateId: string | null,
+  templateId: string,
```

Comment:
This isn't related to the fix - while digging around, I just noticed that this should be `string` instead of `string | null`